### PR TITLE
Fix ruby >= 2 dependencies by pinning back down some fluentd plugins

### LINKED ISSUE #81: fluentd-0.14 requires ruby >= 2
And we are not ready for that upgrade yet, unfortunately

### PR DESCRIPTION
**Pull request by gozer**
Annoyingly, this requires forked GEMs hosted on people.m.o for now.

Fixes #81

### CODE REVIEW

File: nubis/puppet/init.pp [hunk 1]
```diff
@@ -18,28 +23,22 @@
   plugin_name => 'fluent-plugin-sqs',
 }
 
-fluentd::install_plugin { 'aws-elasticsearch-service':
-  ensure      => '0.1.4',
-  plugin_type => 'gem',
-  plugin_name => 'fluent-plugin-aws-elasticsearch-service',
-}->wget::fetch { "patch aws-elasticsearch-service for ruby < 2":
```
Comment (gozer) [inline nubis/puppet/init.pp]: On purpose, this workaround isn't needed anymore.

File: nubis/puppet/init.pp [hunk 2]
```diff
@@ -18,28 +23,22 @@
...
-    source => "https://raw.githubusercontent.com/atomita/fluent-plugin-aws-elasticsearch-service/a823433920bb183dbdda087f3c0fcca6c381bef7/lib/fluent/plugin/out_aws-elasticsearch-service.rb",
-    destination => "/usr/lib/fluent/ruby/lib/ruby/gems/1.9.1/gems/fluent-plugin-aws-elasticsearch-service-0.1.4/lib/fluent/plugin/out_aws-elasticsearch-service.rb",
-    user => "root",
-    verbose => true,
-    redownload => true, # The file already exists, we replace it
-}->wget::fetch { "patch faraday_middleware-aws-signers for ruby < 2":
```
Comment (gozer) [inline nubis/puppet/init.pp]: On purpose, this workaround isn't needed anymore.

File: nubis/puppet/init.pp [hunk 3]
```diff
@@ -18,28 +23,22 @@
...
+    Package["td-agent"],
+  ],
 }
 
-fluentd::install_plugin { 'elasticsearch':
-  ensure      => '1.3.0',
-  plugin_type => 'gem',
-  plugin_name => 'fluent-plugin-elasticsearch',
+package { "fluent-plugin-elasticsearch":
+  ensure   => "1.5.0",
```
Comment (gozer) [inline nubis/puppet/init.pp]: ride-along upgrade is not strictly necessary, but was simpler for me to generate a gem against, so here it is.

**Comment by tinnightcap** [comment]
r+

**Pull request merged**